docs/api.md

```diff
@@ -241,6 +241,10 @@ Content:
                         "description": "Rating from the current client. See 'Get Comment Rating' API",
                         "type": "integer"
                     },
+                    "score": {
+                        "description": "The score of the current comments. Thumbs up minus thumbs down.",
```

Comment:
Maybe something like that would be better?
"Total comment score. Difference between up-votes and down-votes"